source/util/AMSettings.cpp

```diff
@@ -171,9 +170,9 @@ void AMSettings::load() {
 	publicDataFolder_ = settings.value("publicDataFolder", "/home/acquaman/data/").toString();
 	publicDatabaseFilename_ = settings.value("publicDatabaseFilename", "publicdata.db").toString();
 
-	QString defaultBasePath(QDir::homePath() % "/beamline/programming");
-	fileLoaderPluginsFolder_ = settings.value("fileLoaderPluginsFolder", QString(defaultBasePath % "/acquaman/plugins/FileLoaders")).toString();
-	analysisBlockPluginsFolder_ = settings.value("analysisBlockPluginsFolder", QString(defaultBasePath % "/acquaman/plugins/AnalysisBlocks")).toString();
+	QString defaultBasePath(QCoreApplication::applicationDirPath() % "/..");
```

Comment:
@helfrij for SGM/PGM/SXRMB, acquaman is running from `acquaman/bin`; if there is no exception, the other beamlines should run acquaman from `acquaman/build`. Therefore, `QCoreApplication::applicationDirPath()` includes either `acquaman/bin` or `acquaman/build`. That's why we also need to add `/..` for the plugin folder